### ISSUE TITLE
simple concenation results in parsing error, concenate null values as empty string

### ISSUE THREAD

**Issue by doczoidberg** [handsontable/hyperformula#495]
I still have _many_ issues with HF. It is a mess to debug. 
e.g. I have to go through all references here:

`=C1198&";"&D1198&";"&E1198&";"&F1198&";"&G1198&H1198&":"&I1198&":"&J1198&";"&K1198&":"&L1198&":"&M1198&";"&N1198&";"&O1198&";"&P1198&";"&Q1198&";"&R1198&";"&S1198&";"&T1198&";"&U1198&";"&V1198&";"&W1198&";"&X1198&";"&Y1198`

```
DetailedCellError {value: "#ERROR!", type: "ERROR", message: "Parsing error"}
message: "Parsing error"
type: "ERROR"
value: "#ERROR!"
__proto__: Object
```

**Comment by doczoidberg** [comment]
one can't concenate null values.

This should be used like an empty string?

**Comment by wojciechczerniak** [comment]
Should be fixed in #499

**Comment by doczoidberg** [comment]
good work!
